package.json

```diff
@@ -30,6 +30,7 @@
     "react-dates": "15.5.3",
     "rrule": "2.2.0",
     "set-dom": "7.5.2",
+    "terser": "4.0.0",
```

Comment:
@jridgewell I just realized this is in `dependencies`, when it should probably be in `devDependencies`.

Reply:
#24018